apm/docs/metrics.md

```diff
@@ -1,610 +1,961 @@
+
+##### time_usage(s)
+Consensus time for P90
+```
+avg(histogram_quantile(0.90, sum(rate(muta_consensus_duration_seconds_bucket[5m])) by (le, instance)))
+```
+</details>
+
+#### exec_p90
+- description: Consensus exec time for P90
+<details>
+<summary>Legende details</summary>
+
+##### /
+Consensus exec time for P90
+```
+avg(histogram_quantile(0.90, sum(rate(muta_consensus_time_cost_seconds_bucket{type="exec"}[5m])) by (le, instance)))
+```
+</details>
+
+#### put_cf_each_block_time_usage
+- description: Average time to write a block height
+<details>
+<summary>Legende details</summary>
+
+##### /
+Average time to write a block height
+```
+avg (sum by (instance) (increase(muta_storage_put_cf_seconds[5m]))) / avg(increase(muta_consensus_height[5m]))
+```
+</details>
+
+#### get_cf_each_block_time_usage
+- description: Average time to read a block height
+<details>
+<summary>Legende details</summary>
+
+##### /
+Average time to read a block height
+```
+avg (sum by (instance) (increase(muta_storage_get_cf_seconds[5m]))) / avg(increase(muta_consensus_height[5m]))
+```
+</details>
+
+
+#### processed_tx_request
+- description: Monitor transaction requests
+<details>
+<summary>Legende details</summary>
+
+##### Total
+Total number of transaction requests
+```
+sum(rate(muta_api_request_result_total{type="send_transaction"}[5m]))
+```
+
+##### Success Total
+Total number of successful transaction requests
+```
+sum(rate(muta_api_request_result_total{result="success",type="send_transaction"}[5m]))
+```
+
+##### instance
+Number of successful transaction requests sent by nodes
+```
+rate(muta_api_request_result_total{result="success", type="send_transaction"}[5m])
+```
+</details>
+
+#### current_height
+- description: Chain current height
+<details>
+<summary>Legende details</summary>
+
+##### {{instance}}
+Node current height
+
+```
+sort_desc(muta_consensus_height)
+```
+</details>
+
+#### synced_block
+- description: Number of blocks synchronized by nodes
+<details>
+<summary>Legende details</summary>
+
+##### {{instance}}
+Number of blocks synchronized by nodes
+```
+muta_consensus_sync_block_total 
+```
+</details>
+
+
+#### network_message_arrival_rate
+- description: Estimate the network message arrival rate in the last five minutes
+<details>
+<summary>Legende details</summary>
+
+##### /
+Estimate the network message arrival rate in the last five minutes
+```
+((sum(muta_network_message_total{target="all", direction="sent"}) * 20) +
+ (sum(muta_network_message_total{target="single", direction="sent"})) )
+/
+sum(muta_network_message_total{direction="received"}) 
+
+```
+</details>
+
+
+#### consensus_round_cost
+- description: Number of rounds needed to reach consensus
+<details>
+<summary>Legende details</summary>
+
+##### {{instance}}
+Number of rounds needed to reach consensus
+```
+(muta_consensus_round > 0 )
+```
+</details>
+
+#### mempool_cached_tx
+- description: Number of transactions in the current trading pool
```

Comment:
Number of transactions in the current mempool

the mempool is the name of Muta's component, use of uniform naming to avoid confusion